PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -467,10 +467,7 @@ INT_PTR CALLBACK Shortcut::run_dlgProc(UINT Message, WPARAM wParam, LPARAM)
 // return true if one of CommandShortcuts is deleted. Otherwise false.
 void Accelerator::updateShortcuts() 
 {
-	vector<int> incrFindAccIds;
-	incrFindAccIds.push_back(IDM_SEARCH_FINDNEXT);
-	incrFindAccIds.push_back(IDM_SEARCH_FINDPREV);
-	incrFindAccIds.push_back(IDM_SEARCH_FINDINCREMENT);
+	const unsigned long incrFindAccIds[] = { IDM_SEARCH_FINDNEXT, IDM_SEARCH_FINDPREV, IDM_SEARCH_FINDINCREMENT };

```

Comment:
should be possible to use std:array here instead of falling back to plain old c array

Reply:
You are absolutely right, I actually realized this as I reviewed it. Might as well... The only issue I have with it is that I have to hardcode the amount of entries, so must be done like

```std::array<unsigned long, 3> incrFindAccIds = { a, b, c };```

whereas with plain C array I don't need to do it.

Reply:
The solution would be http://en.cppreference.com/w/cpp/experimental/make_array, but to my feeling adding the size is still ok. Also because the compiler will detect if the initlist doesn't match with specified array size.